.github/workflows/release-snapshot.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
           VERSION=$(./gradlew :core:properties -q | grep "version:" | awk '{print $2}')
           echo "version=${VERSION}" >> $GITHUB_OUTPUT
 
-      - name: Upload SNAPSHOT versions for scalardb, scalardb-schema-loader, and scalardb-integration-test to Maven Snapshot Repository
+      - name: Upload SNAPSHOT versions for scalardb, scalardb-schema-loader, scalardb-data-loader-core, and scalardb-integration-test to Maven Snapshot Repository
```

Comment:
Could you please make the same change for the follows:
https://github.com/scalar-labs/scalardb/blob/5583eabe6cf6f315cbbc44bec7509f295069989c/.github/workflows/upload-artifacts.yaml#L60